modules/stampcollection.py

```diff
@@ -58,6 +74,38 @@ def update_vote(self, emoji: str, from_id: int, to_id: int,
         if recalculate:
             self.calculate_stamps()
 
+    def update_stamps_in_users_table(
+        self, user: DiscordUser, stamp_count: float
+    ) -> None:
+        """Update stamps count in Coda [users/team table](https://coda.io/d/AI-Safety-Info_dfau7sl2hmG/Team_sur3i#_lu_Rc)"""
+        query = f'"Discord handle":"{get_user_handle(user)}"'
+        user_row = utils.get_user_row(query)
+        if user_row is None:
+            return
+        user_id = user_row["id"]
+        uri = f"https://coda.io/apis/v1/docs/{utils.DOC_ID}/tables/{utils.TEAM_GRID_ID}/rows/{user_id}"
+        payload = {
+            "row": {
+                "cells": [
+                    {"column": "Stamp count", "value": stamp_count},
+                ],
+            },
+        }
+        response = requests.put(
+            uri, headers=utils.get_coda_auth_headers(), json=payload, timeout=20
+        )
+        response.raise_for_status()
+        log.info("Updated %s's stamp count to %f", user.name, stamp_count)
```

Comment:
```suggestion
        log.info(self.class_name, "Updated %s's stamp count to %f on coda", user.name, stamp_count)
```

Reply:
I don't this (putting `self.class_name` before the main string) is going to work, unless `structlog`'s logging works differently than standard python's logging. What should it output?

Reply:
i'm happy with your latest update to logging. it is different to how other ppl have done logging. i am new to all this so have no strong opinions, but what you did does look a bit cleaner:

<img width="645" alt="image" src="https://user-images.githubusercontent.com/25344832/222903905-fa0f9b89-6fd0-4bdf-9dea-01fec75a837d.png">

Reply:
oh, wait, I just checked how most logging is done in this codebase and what you suggested was correct
yeah, it looks cleaner but I'll probably revert to what you suggested, although AFAIR Rob suggested to change the way we do logging at some point

Reply:
i think we need:

`log.info(self.class_name, msg="...",)`